src/manifest-schema.json

```diff
@@ -141,6 +141,80 @@
         }
       }
     },
+    "provider": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "ja": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "name": {
+              "type": "string",
+              "minLength": 1,
+              "maxLength": 100
+            },
+            "url": {
+              "type": "string",
+              "minLength": 1,
+              "format": "https-url"
+            }
+          }

```

Comment:
"name" and "url" are required when specify provider property. 
Refer: https://sharedoc.atlassian.net/wiki/spaces/grn/pages/2457174040/GTM-9522+Garoon+plugin+Add+provider+property+to+plugin+s+information+of+manifest.json
So, please add this code after this line
"required": [
            "name",
            "url"
          ]